packages/dependency-extraction-webpack-plugin/index.js

```diff
@@ -98,27 +110,31 @@ class DependencyExtractionWebpackPlugin {
 					}
 				}
 
-				// Build a stable JSON string that declares the WordPress script dependencies.
-				const sortedDepsArray = Array.from( entrypointExternalizedWpDeps ).sort();
-				const depsString = JSON.stringify( sortedDepsArray );
+				const runtimeChunk = entrypoint.getRuntimeChunk();
+
+				// Build a stable JSON string that declares the WordPress script asset.
+				const assetString = this.serialize( {
+					dependencies: Array.from( entrypointExternalizedWpDeps ).sort(),
+					version: runtimeChunk.hash,
+				} );
 
-				// Determine a filename for the `[entrypoint].deps.json` file.
+				// Determine a filename for the `[entrypoint].asset.[outputFormat]` file.
```

Comment:
Maybe we shouldn't be so specific? The README can cover what the expected filenames are.

```suggestion
				// Determine a filename for the asset file.
```